virt/virtual_machines/vm_networking/virt-viewing-ip-of-vm-nic.adoc

```diff
@@ -4,17 +4,7 @@ include::modules/virt-document-attributes.adoc[]
 :context: virt-viewing-ip-of-vm-vnic
 toc::[]
 
-The QEMU guest agent runs on the virtual machine and passes the IP address of attached NICs to the host, allowing you to view the IP address from both the web console and the `oc` client.
-
-== Prerequisites
-
-. Verify that the guest agent is installed and running by entering the following command:
-+
-----
-$ systemctl status qemu-guest-agent
-----
-+
-. If the guest agent is not installed and running, xref:../../../virt/virtual_machines/vm_networking/virt-installing-qemu-guest-agent.adoc#virt-installing-qemu-guest-agent[install and run the guest agent on the virtual machine].
+You can view the IP address for a Network Interface Card (NIC) by using the web console or the `oc` client. The xref:../../../virt/virtual_machines/virt-installing-qemu-guest-agent.adoc#virt-installing-qemu-guest-agent[QEMU guest agent] displays additional information about the virtual machine, users, file systems, and secondary networks,
```

Comment:
There is comma instead of full stop :eyes: 

Also, I don't think that  "users, file systems" is relevant here. Can we just say it shows additional info about VM's secondary networks?

Reply:
Yes, content was updated to remove users, file systems.

Reply:
@phoracek - please review.